global.js

```diff
@@ -57,6 +58,10 @@ window.onload = function() {
   var submitButton = document.getElementById("submitter");
   submitButton.addEventListener("click", function () {
     // need to add a function that does something with the user's answer.
+    var questionResult = getElementById("question_result");
```

Comment:
When Submit is clicked, what is the state of the DOM? Is there an element with the id "question_result" in it? If its style display is set to none, is it in the DOM to manipulate?